crates/proof-of-sql/src/sql/proof_exprs/comparison_util.rs

```diff
@@ -83,13 +78,7 @@ pub(crate) fn scale_and_subtract<'a, S: Scalar>(
             rhs_precision_value + (max_scale - rhs_scale) as u8,
         );
         // Check if the precision is valid
-        let _max_precision = Precision::new(max_precision_value).map_err(|_| {
-            ConversionError::DecimalConversionError {
-                source: DecimalError::InvalidPrecision {
-                    error: max_precision_value.to_string(),
-                },
-            }
-        })?;
+        let _max_precision = Precision::new(max_precision_value)?;
```

Comment:
why is this unused? would it be better to have a check function that didnt require an unused variable?